### PR TITLE
Further backend changes for #48

### PR DESCRIPTION
**Pull request by roberthampton**
Sends a quiz object on subscription to event stream. Should address bug where lobby doesn't update until someone else joins.

### CODE REVIEW

File: Back-end/NodeAPI/index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <script>
     //Example front end code, showing how to create event source from api and add an eventListener to this event source.
     //Replace the participant ID below with users participant ID
-    var serverEvents = new EventSource('http://35.214.82.56:3000/stream/5f80387b5cd3bf1f70003d8a');
```
Comment (khanjason) [inline Back-end/NodeAPI/index.html:4]: Why was this changed to localhost

Comment (roberthampton) [reply Back-end/NodeAPI/index.html:4]: That was just as part of testing, it doesn't matter or have an effect on the server as this file is just for testing event streams, and it probably makes sense for testing changes now as they will be running locally first as we already know the server is working, but I could change it back if you'd prefer?

Comment (khanjason) [reply Back-end/NodeAPI/index.html:4]: It's fine for now, it can be changed back later

**Review by khanjason — APPROVED** [review]
changes look better, cleaner code

**Review by raufnawaztarar — APPROVED** [review]
